internal/init/init.go

```diff
@@ -156,3 +166,22 @@ func writeVscodeConfig(fsys afero.Fs) error {
 	fmt.Println("Generated VS Code settings in " + utils.Bold(settingsPath) + ". Please install the recommended extension!")
 	return nil
 }
+
+func writeIntelliJConfig(fsys afero.Fs) error {
+	if err := utils.MkdirIfNotExistFS(fsys, intellijDir); err != nil {
+		return err
+	}
+	if err := createDenoConfig(denoPath, intelliJDeno, fsys); err != nil {
+		return err
+	}
+
+	fmt.Println("Generated IntelliJ settings in " + utils.Bold(denoPath) + ". Please install the Deno plugin!")
+	return nil
+}
+
+func createDenoConfig(path string, template string, fsys afero.Fs) error {
+	if err := afero.WriteFile(fsys, path, []byte(template), 0644); err != nil {
+		return err
+	}
+	return nil
```

Comment:
```suggestion
	return afero.WriteFile(fsys, path, []byte(template), 0644)
```

There's also utils.WriteFile